ranged_weapon.h

```diff
@@ -17,6 +17,7 @@
 
 #include "util.h"
 #include "item.h"
+#define MAX_RANGED_DISTANCE 10
```

Comment:
How about defining this as a variable inside RangedWeapon.

This way this could changed in item_type.cpp for various weapons. The way to change it could be similar to how various furniture parameters are changed, for example: RangedWeapon(...) // constructor sets 10 by default
     .setMaxDistance(123); // override it like this